Entities/Items/Crate/Crate.as

```diff
@@ -162,6 +185,36 @@ bool doesCollideWithBlob(CBlob@ this, CBlob@ blob)
 	return !blob.hasTag("parachute");
 }
 
+void onCollision( CBlob@ this, CBlob@ blob, bool solid ){
+	
+	if(!isServer())return;
```

Comment:
Add spaces between `)` and other tokens, also avoid the `f( a, b, c );` "convention" for Crimson, rather do `f(a, b, c);`

Reply:
Sorry, came from the manual, or somewhere